src/plugins/oSnap/components/TransactionBuilder/TransferFunds.vue

```diff
@@ -55,10 +54,10 @@ function updateTransaction() {
       recipient: recipient.value,
       token: selectedToken.value
     });
+    emit('updateTransaction', transaction);
     const isTransactionValid = validateTransaction(transaction);
-    if (isTransactionValid) {
-      emit('updateTransaction', transaction);
-      return;
+    if (!isTransactionValid) {
+      console.warn('invalid transaction');
     }
   } catch (error) {
```

Comment:
what happens here? if transaction errors the parent never sees this is invalid

Reply:
can remove conditional entirely